holmes/main.py

```diff
@@ -1,10 +1,10 @@
 # ruff: noqa: E402
 import os
-from holmes.core.prompt import append_file_to_user_prompt
 from collections import OrderedDict
```

Comment:
Can you please move all the imports after line 16?

This code, loads custom certificates (it's used by some users that runs Holmes on environments with Proxies with self signed certificates)

Doing imports before loading the certificates, might indirectly start some http client, without the custom certificate, which will cause it to fail on those environments.
So we add imports, only after loading the custom certificates, if exists